pennylane/drawer/tape_text.py

```diff
@@ -244,7 +248,7 @@ def _add_measurement(m, layer_str, config):
     return layer_str
 
 
-# pylint: disable=too-many-arguments
+# pylint: disable=too-many-arguments, too-many-positional-arguments
```

Comment:
Maybe time to add a `,*,` to the signature? Maybe after `wire_order`?

Reply:
:heavy_check_mark: